rpcd/playbooks/roles/rpc_support/templates/holland.conf.j2

```diff
@@ -16,7 +16,8 @@ backupsets = rpc_support
 umask = 0007
 
 # Define a path for holland and its spawned processes
-path = /usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin
+
+path = {% if holland_venv_enabled | bool %}{{holland_venv_bin}}:{% endif %}/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin
```

Comment:
Added the venv to the PATH would allow the user to globally invoke holland, but it would not run within the venv